test/contracts-network/colony-reward-payouts.js

```diff
@@ -514,7 +514,7 @@ contract("Colony Reward Payouts", (accounts) => {
 
       await checkErrorRevert(
         colony.claimRewardPayout(payoutId, initialSquareRoots, ...userReputationProof1, { from: userAddress1 }),
-        "colony-reward-payout-lock-count-too-high"
+        "colony-token-already-unlocked"
```

Comment:
Mm this seems like the wrong error message -- don't we want `colony-token-locking-has-previous-active-locks`?

Reply:
What about this?

```
    require(lockCountDelta <= 1, "colony-token-locking-has-previous-active-locks");
    require(lockCountDelta != 0, "colony-token-already-unlocked");
```

The first one will trigger for all diffs except 0 and 1, indicating that there are existing locks. The second then triggers only in the remaining case of a diff of 0, indicating the token is unlocked. If both fail to trigger, it means we have 1, the correct value.

Reply:
> Mm this seems like the wrong error message -- don't we want colony-token-locking-has-previous-active-locks?

I am definitely not up to scratch today, but I don't think so? When this error message is being checked, we have already waived the payout, and we're checking that we can't claim a payout that we've waived. So there should not be previous active locks, otherwise we've not correctly waived this payout.

> Seems like the issue is that the requires as they are currently can't differentiate between a diff of 0 and a diff of >1.

The requires are currently:
```
    require(lockCountDelta != 0, "colony-token-already-unlocked");
    require(lockCountDelta == 1, "colony-token-locking-has-previous-active-locks");
```
So surely it can differentiate? If the diff is 0, the first one triggers and if the diff is >1, the second one triggers. I think I must just not be understanding something!

Reply:
Ah my mistake, I didn't see the entire test in the diff view and for some reason I assumed these tests were for locked tokens. You're right, looks like things are fine as they are.